test/compilable/test11259.d

```diff
@@ -0,0 +1,6 @@
+// https://issues.dlang.org/show_bug.cgi?id=11259
+
+import imports.imp11259;
+
+static assert(__traits(isSame, __traits(parent, imports.imp11259), imports));
+static assert(__traits(isSame, imports, __traits(parent, imports.imp11259)));
```

Comment:
Can you add a test for nested packages, e.g:
```D
version (Posix)
{
    // smallest druntime module without imports on posix
    import core.sys.posix.libgen;
    static assert(__traits(isSame, __traits(parent, core.sys.posix.libgen), core.sys.posix));
    static assert(__traits(isSame, __traits(parent, core.sys.posix), core.sys));    
}
else
{
    // smallest module without imports for windows
    import core.sys.windows.lmuseflg;
    static assert(__traits(isSame, __traits(parent, core.sys.windows.lmuseflg), core.sys.windows));
    static assert(__traits(isSame, __traits(parent, core.sys.windows), core.sys));
}

static assert(__traits(isSame, __traits(parent, core.sys), core));
```

(Using phobos in the test-suite is discouraged, but I think druntime is ok.)